charts/k8s-watcher/templates/daemonset-metrics.yaml

```diff
@@ -0,0 +1,62 @@
+{{- if .Values.metrics.enabled }}
+apiVersion: apps/v1
+kind: DaemonSet
```

Comment:
do we really have to install it as a daemonset on clients? won't a deployment suffice for this matter?

Reply:
we wanted to support large costumers 
so we decided to use Daemonset